### ISSUE TITLE
shadow-clj support

### ISSUE THREAD

**Issue by ZabelTech** [viebel/klipse-clj#8]
Hi @viebel , 
I'm coming over from https://github.com/viebel/klipse/issues/353 (plugging in shadow-cljs support)
Shadow-cljs intentionally hardcodes the compiler flag `dump-core` to false, that means the analyzer-cache will be empty initially and we need to put in `cljs.core` by hand.
For this I need you to put the analyzer cache for `cljs.core` up next to `core$macros.cljc.cache.json` in the `bundled-ns-root`. 
I can make the rest happen then, sounds good?

**Comment by viebel** [comment]
I don't understand what you mean by "to put the analyzer cache for cljs.core up next to core$macros.cljc.cache.json in the bundled-ns-root."
Could you open a PR for it?

**Comment by ZabelTech** [comment]
@viebel Sure, there will be a PR.

But what I was trying to say above is: 
To get it going with shadowcljs the analyzer cache for cljs.core needs to be hosted. Just like you allready do it with the marcos.

Here is a somewhat lengthy discussion about the technical details: https://github.com/thheller/shadow-cljs/issues/922

Tldr:  Thats currently the only possible solution and it's pretty likely the best one from a ux perspective.

So, the deal would be: 
You put the analyzer cache of cljs.core where you host the other 'bundled' namespace caches.
I'm going to handle the Code.
You will get a PR and than we close the long standing issue about the shadowcljs support.
Sounds good?

**Comment by viebel** [comment]
Sounds perfect @ZabelTech !
In fact, the analyzer cache for cljs.core is already hosted.
The hosting of analyzer cache is handled in this repo https://github.com/viebel/cljs-analysis-cache.
The hosted files come from this folder https://github.com/viebel/cljs-analysis-cache/tree/master/cache

Let me know if you need further explanation.

**Comment by ZabelTech** [comment]
thanks @viebel, I have everything that I need for now.
